### DISCUSSION TITLE
VirtualList组件支持itemSize 可变高度？

### DISCUSSION THREAD

**Discussion by ziqiudev** [NervJS/taro#13088] [Q & A] [answered]
### 这个特性解决了什么问题？
希望VirtualList支持 列表节点可变高度
需求通常是节点高度是可变的

### 这个 API 长什么样？
itemSize支持函数，来动态的设置列表节点高度

**Comment by ZakaryCode** [comment, marked as answer]
这是一个已支持的特性，3.6 版本以下，在 React 项目可以通过 unlimitedSize 使用节点真实高度； v3.6+ 可以在 React、Vue 项目中使用 unlimitedSize 或者传入函数 itemSize 动态设置。

**Comment by ShawnZeng** [comment]
设置了unlimitedSize， 并不起作用。而且动态设置itemSize的函数有个Bug。index = 1抓不到。index = 2以上才能捕捉到。

**Reply by jiajiaxing1** [reply]
你是多少版本呀，我taro版本3.5.0，@tarojs/components-advanced是3.6.36，目前设置了unlimitedSize=true，确实不起作用，列表会重叠，itemSize传入函数打印出来的index和data都为null
